orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplier.java

```diff
@@ -0,0 +1,51 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.orca.clouddriver.OortService;
+import com.netflix.spinnaker.orca.kato.pipeline.support.StageData;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import retrofit.client.Response;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Arrays;
+
+@Slf4j
+@Component
+public class TitusInterestingHealthProviderNamesSupplier implements InterestingHealthProviderNamesSupplier {
+  private static final String TITUS = "Titus";
+  private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
+  private static final String ROLLING_PUSH = "rollingpush";
+  private static final List<String> SUPPORTED_STAGES = Arrays.asList("deploy", "enableServerGroup", "createServerGroup", "cloneServerGroup", "disableServerGroup");
+
+  @Override
+  public boolean supports(String cloudProvider, Stage stage) {
+    return cloudProvider.equalsIgnoreCase(TITUS) && SUPPORTED_STAGES.contains(stage.getType()) && stage.getContext().containsKey("strategy") && stage.getContext().get("strategy").equals(ROLLING_PUSH);
+  }
+
+  @Override
+  public List<String> process(String cloudProvider, OortService oortService, ObjectMapper objectMapper, Stage stage) {
```

Comment:
IMO `oortService` and `objectMapper` should be `@Autowire` into the constructor of this class rather than pollute a public constructor.